compiler/src/iree/compiler/Codegen/Dialect/LoweringConfig.td

```diff
@@ -36,21 +36,22 @@ def LLVMGPU_MatmulSimt : I32EnumAttrCase<"LLVMGPUMatmulSimt", 11>;
 def LLVMGPU_MatmulTensorCore : I32EnumAttrCase<"LLVMGPUMatmulTensorCore", 12>;
 def LLVMGPU_TransposeSharedMem : I32EnumAttrCase<"LLVMGPUTransposeSharedMem", 13>;
 def LLVMGPU_WarpReduction : I32EnumAttrCase<"LLVMGPUWarpReduction", 14>;
+def LLVMGPU_DataTiling : I32EnumAttrCase<"LLVMGPUDataTiling", 15>;
```

Comment:
why is it call data tiling? This is just the packing lowering. Let's not mix the two.

Reply:
This is the pipeline for data tiling relayout lowering at this moment. The pack and unpack op lowering is going to use this pipeline, though they might get diverged once we have more study.

Reply:
why is a pass pipeline related to data tiling relayout? pass pipelines are linked to the kind of op they lower and the associated transformation sequence. relayout may generate different kind of ops I think it should be orthogonal to the lowering of those ops. I would like to understand this part as this feels very counter intuitive to me.

Reply:
I see your point. The intention of adding a new pipeline for pack/unpack ops is that we're in a fast development mode. We could reuse other pipelines for them in the future. Since we're not clear about how to lower those ops efficiently, I'd prefer having a new pipeline for them. What about renaming it to `LLVMGPUPackUnPackLoweroing`?

Reply:
You would need a fall back of handling dispatches that have only pack/unpack operations... that is required for basic correctness/coverage. If not its very easy to fall into this crack.

Reply:
Yes using a new pipeline if fine. I was mostly pointing out the naming that doesn't make sense to me. `LLVMGPUPack` kind of name makes sense to me.

Reply:
Ideally you would have one pipeline for all operations that implement the `TilingInterface`. Then we dont need to add new pipelines per op for the fall back case.